core/sql/executor/SequenceFileReader.cpp

```diff
@@ -189,9 +192,10 @@ SFR_RetCode SequenceFileReader::open(const char* path)
   if (jresult != NULL)
   {
     logError(CAT_SQL_HDFS_SEQ_FILE_READER, "SequenceFileReader::open()", jresult);
+    jenv_->PopLocalFrame(NULL);
```

Comment:
The DeleteLocalRef at line 190 is no longer needed, but harmless, right?

Reply:
Yes. Will remove